keyboards/claw44/keymaps/default/rules.mk

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
I think this whole file's contents should be in `keyboards/claw44/rev1/rules.mk` instead, and this file either zero-length or deleted entirely.

Keymap-level `rules.mk` files should be used by users for overriding the default settings that are set at the keyboard level.